tests/test_basic.py

```diff
@@ -182,8 +182,6 @@ def test_osd_number(rook_cluster):
         i += 1
 
 
-# FIXME
-@pytest.mark.skip(reason="This is currently unstable")
```

Comment:
Still failing but maybe mark it as `@pytest.mark.xfail` it it's executed but expected to fail?